external_executors/local_cromwell/run_portability_test.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""A script to run a WDL (and its subWDLs) in a local cromwell environment.
+
+This is placed within a docker container that has cromwell and tools for
+localizing inputs installed. It's run as the entrypoint to the container.
+
+There is one significant and very sad constraint on how this script can be
+run. This is being run in a docker container, and the WDL it executes can
+specify docker images in which to run tasks. So we've got a
+docker-within-docker situation. And, the docker socket available to this
+script and to cromwell is mounted from the calling environment, so you
+have to make sure any paths you try to mount are valid paths in the calling
+environment, not within this container.
+"""
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+
+# This script is meant to be run within a docker container, and it's going to
+# have to assume that an external directory has been mounted somewhere. That
+# location is set here.
+IO_DIR = "/io"
```

Comment:
os.sep is safer/more portable, just incase.

Reply:
Yep, fixed.